standalone/include/psen_scan_v2_standalone/data_conversion_layer/io_state_conversions.h

```diff
@@ -87,6 +87,40 @@ static inline std::vector<PinState> generateOutputPinStates(const monitoring_fra
   return pin_states;
 }
 
+static inline std::vector<PinState> generateChangedInputStates(const monitoring_frame::io::PinData& new_state,
+                                                               const monitoring_frame::io::PinData& old_state)
+{
+  std::vector<PinState> pin_states;
```

Comment:
Maybe add reserve here since the size is known in advance

Reply:
Do we know? We know the max size which will much to big in almost all cases.
I think the size can be anything between 1 and I think its about 21 actual used input bits at the moment?

Reply:
Yep your right